### PR TITLE
INTG-270 Exclude inactive fields when exporting CSV

### PR DESCRIPTION
**Pull request by tonyOreglia**
Added the option for users to exclude inactive items from the CSV export. 
Items that are nested under a smartfield are inactivated if the smartfield condition is false.

The option can be toggled via the config.yaml file. i.e. config.yaml: 
```
API:
    token: YOUR_API_TOKEN
export_options:
    export_path:
    timezone:
    filename: 
    csv_options:
        export_inactive_items: false
```

Also, added unit tests around this functionality. From test_csv_exporter.py:
```
def test_export_inactive_fields_true(self):
        csv_exporter = csv.CsvExporter(json.load(open(os.path.join(self.path_to_test_files, 'export_inactive_fields_true.json'), 'r')), True)
        csv_exporter.save_converted_audit_to_file('test 36.csv', allow_overwrite=True)
        self.assertEqual(open('test 36.csv', 'r').read(), open(os.path.join(self.path_to_test_files, 'export_inactive_fields_true_expected_output.csv'), 'r').read())
        os.remove('test 36.csv')
```

```
def test_do_not_export_inactive_fields(self):
        csv_exporter = csv.CsvExporter(json.load(open(os.path.join(self.path_to_test_files, 'do_not_export_inactive_fields.json'), 'r')), False)
        csv_exporter.save_converted_audit_to_file('test 37.csv', allow_overwrite=True)
        self.assertEqual(open('test 37.csv', 'r').read(), open(os.path.join(self.path_to_test_files, 'do_not_export_inactive_fields_expected_output.csv'), 'r').read())
        os.remove('test 37.csv')
```

### CODE REVIEW

File: tools/exporter/ReadMe.md [hunk 1]
```diff
@@ -99,6 +98,7 @@ Things you can configure:
 | filename  |  an audit item ID whose response is going to be used to name the files of exported audit reports. Can only be an item with a response type of `text` from the header section of the audit such as Audit Title, Document No., Client / Site, Prepared By, Personnel, or any custom header item which has a 'text' type response |
 | export_profiles  | to apply an export profile transformation to particular templates, give here a list of export profile ids
 | sync_delay_in_seconds | time in seconds to wait after completing one export run, before running again
+| export_inactive_items | This setting only applies when exporting to CSV. Valid values are true (export all items) or false (do not export inactive items). Items that are nested under [Smart Field](https://support.safetyculture.com/templates/smart-fields/) may be inactive.
```
Comment (dimitrikoubaroulis) [inline tools/exporter/ReadMe.md]: may be inactive => will be 'inactive' if the smart field condition is not satisfied for these items

File: tools/exporter/config.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 API:
-    token: YOUR_SAFETYCULTURE_API_TOKEN
+    token: YOUR_API_TOKEN
```
Comment (dimitrikoubaroulis) [inline tools/exporter/config.yaml]: Why is this better?

File: tools/exporter/tests/create_unit_test.py [hunk 3]
```diff
@@ -1,68 +0,0 @@
-import sys
```
Comment (dimitrikoubaroulis) [inline tools/exporter/tests/create_unit_test.py]: Why was this removed?

File: tools/exporter/tests/test_csv_exporter.py [hunk 4]
```diff
@@ -215,6 +215,18 @@ def test_single_question_yes___no___na_comment_and_images_added_to_question(self
         self.assertEqual(open('test 35.csv', 'r').read(), open(os.path.join(self.path_to_test_files, 'unit_test_single_question_yes___no___na_comment_and_images_added_to_question_expected_output.csv'), 'r').read())
         os.remove('test 35.csv')
 
+    def test_export_inactive_fields_true(self):
```
Comment (dimitrikoubaroulis) [inline tools/exporter/tests/test_csv_exporter.py]: Nice to see new functionality covered by unit tests!

File: tools/exporter/tests/create_unit_test.py (hunk 3 above)
Comment (tonyOreglia) [inline tools/exporter/tests/create_unit_test.py]: Code is messy. It does not have comments or error handling. If a customer decides to use it, I think it would cause more questions and confusion than value.
